library/cmac.c

```diff
@@ -779,6 +779,8 @@ static int cmac_test_subkeys( int verbose,
 
         if( verbose != 0 )
             mbedtls_printf( "passed\n" );
+
+        mbedtls_cipher_free( &ctx );
```

Comment:
I believe that there is a possibility of a double free. When all loop iterations are completed, the `ctx` would have been  freed, but the `mbedtls_cipher_free()` at the exit label will be executed anyways.